### ISSUE TITLE
[SF][SegmentedButton][A11y]: [Screen reader does not announce selected state for visually selected segmented button]

### ISSUE THREAD

**Issue by tandan0813** [UI5/webcomponents#13222]
### Describe the bug

When a SegmentedButton has an item initially selected, the selected state should be announced/read out. Currently, this is not happening.

### Isolated Example

https://ui5.github.io/webcomponents-react/v2/?path=/docs/inputs-segmentedbutton--docs&args=accessibleName:Segmented+Button+Example;selectionMode:Single

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

JAWS should announce the selected state of the segmented button.

### UI5 Web Components for React Version

2.18.0

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by kskondov** [comment]
Hello @UI5/ui5-team-ctr-bal,
 Can you please take a look at this issue regarding the segmented button?

Best Regards,
Konstantin

**Comment by tsanislavgatev** [comment]
Hello @tandan0813 ,

Thank you for your issue.

Actually, the case you have reported is currently a known issue and has been reported to the JAWS repository. For a more detailed explanation and discussion on this issue, you can visit the following link: https://github.com/FreedomScientific/standards-support/issues/440

And also for now it does announce the items that are not selected as not selected, so the only one without additional announcement is the selected one.

Please do not hesitate to share any other issues you might encounter.

Kind Regards,
Tsanislav
